AntonBabunin/src/main/java/ua/artcode/market/models/AbstractProduct.java

```diff
@@ -0,0 +1,55 @@
+package ua.artcode.market.models;
+
+public abstract class AbstractProduct {
```

Comment:
Why abstract?

Reply:
Because in classes that inherit from it, some methods must be implemented. Already made them abstract.